lib/smart_answer_flows/next-steps-for-your-business/outcomes/results.erb

```diff
@@ -23,10 +23,11 @@
       border_top: 2,
       padding: true
     } %>
-    <% topics.each do |topic, results| %>
+    <% topics.each_with_index do |(topic, results),index| %>
```

Comment:
Nit: If you like you can do:
```
topics.each.with_index(1) do |(topic, results), index|
```
to [offset the index by 1](https://ruby-doc.org/core-2.7.0/Enumerator.html#method-i-with_index), without having to `+ 1` it later.